fossunited/scheduled_tasks.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+
+import frappe
+
+from fossunited.doctype_ids import EVENT
+
+
+def conclude_events():
+    """
+    Get all the events which have ended (end_date < today) and set their status to concluded
+    """
+    events = frappe.db.get_all(
+        EVENT,
+        {"status": "Live", "event_end_date": ["<", datetime.today()]},
```

Comment:
note : there are events in the platform that don't have an `event_end_date`. I think it isn't a mandatory attribute on the doctype so external events don't seem to have them all the time. Additionally, the old/migrated events don't seem to have this attribute. What happens to those events?

I think that this function will leave those events alone. Right?

Reply:
Yes, it will not fetch those